src/static/containers/AdminCardAdd.jsx

```diff
@@ -67,15 +72,29 @@ class AdminCardAdd extends Component {
     }
 
     render() {
-        const { isFetchingCard, form } = this.props;
+        const { isFetchingCard, form, cardData } = this.props;
         let cardForm;
-        // Update when we link the fetch!
-        const initialData = {};
+        let initialData = {};
         if (this.state.editing) {
-            cardForm = generateForm(form, this.createCard, cardFields, initialData, 'Edit card!');
-        } else {
-            cardForm = generateForm(form, this.createCard, cardFields, initialData, 'Create card!');
+            initialData = {
```

Comment:
How about using react-form and initial values, thus we might need this: https://github.com/zhdmitry/redux-form-antd? And https://stackoverflow.com/a/45495643/1450754 for component not available in `redux-form-antd`.
Yeah I know that means a big refact :+1:

Reply:
Or maybe for a future enhancement, since it's not really in the scope of this issue and not necessary for it to work.

Reply:
I'm not sure about using another lib which gives us not so much:
Some custom components we can create easily and the initial values will stay the same.
I considered passing the data directly to the generateForm in order to hide a little bit the split in initialData but...

Reply:
To be debated elsewhere :)
But I agree for now, let's get it working like that.